src/acl/Checklist.h

```diff
@@ -152,9 +154,12 @@ class ACLChecklist
         return old;
     }
 
+    /// remember the name of the last ACL being evaluated
+    void setLastCheckedName(const SBuf &name) { lastCheckedName_ = name; }
+
 private:
     /// Calls non-blocking check callback with the answer and destroys self.
-    void checkCallback(Acl::Answer answer);
+    void checkCallback(const Acl::Answer &answer);
```

Comment:
This feels out of scope for the current PR

Reply:
IMO, it is in scope because this PR makes Acl::Answer copies a bit more costly; passing Acl::Answer by value is no longer the best or preferred default way. This change avoids one such copy. There is more information about it in PR description:

> Most of these changes are performance-neutral or -positive because they eliminate or reduce memory allocations and associated name copying (and more reduction will become possible after upgrading squid.conf parsers to use SBuf). This change adds SBuf object copies when Acl::Answer is propagated to ACLCB callbacks, but those read-only copies are cheap.

If you insist, we will remove this change from this PR.

Reply:
> If you insist, we will remove this change from this PR.

@kinkie, do you insist on this removal?

Reply:
I don't insist